content/post/deploy-mssql-server-in-aks/index.md

```diff
@@ -0,0 +1,454 @@
+mssqlserver-ag-cluster-secondary   ClusterIP   10.128.213.243   <none>        1433/TCP   9m43s
+```
+
+From the above list, the `mssqlserver-ag-cluster-auth` Secret contains the admin-level credentials needed to connect to the database. Use the following commands to obtain the username and password:
+
+```bash
+$ kubectl get secret -n demo mssqlserver-ag-cluster-auth  -o jsonpath='{.data.username}' | base64 -d
+sa
+$ kubectl get secret -n demo mssqlserver-ag-cluster-auth  -o jsonpath='{.data.password}' | base64 -d
+pT42K58uKEe3pote
+```
+
+
+### Insert Sample Data
+
+In this section, we are going to login into our MSSQL Server pod and insert some sample data.
+
+```bash
+$ kubectl exec -it mssqlserver-ag-cluster-0 -n demo bash
+Defaulted container "mssql" out of: mssql, mssql-coordinator, mssql-init (init)
+mssql@mssqlserver-ag-cluster-0:/$ /opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P "pT42K58uKEe3pote"
+
+1> SELECT name FROM sys.databases
+2> GO
+name                                                                                                                            
+--------------------------------------------------------------------------------------------------------------------------------
+master                                                                                                                          
+tempdb                                                                                                                          
+model                                                                                                                           
+msdb                                                                                                                            
+music                                                                                                                           
+kubedb_system                                                                                                                   
+
+(6 rows affected)
+1> USE music
+2> GO
+Changed database context to 'music'.
+
+1> CREATE TABLE Playlist (Artist NVARCHAR(255), Song NVARCHAR(255));
+2> GO
+1> INSERT INTO Playlist(Artist, Song) VALUES ('John Denver', 'Country Roads');
+2> GO
+
+(1 rows affected)
+1> SELECT * FROM Playlist
+2> GO
+
+Artist                                                                          Song
+---------------------------------------------------------------------------------------------------------------------------------
+---------------------------------------------------------------------------------------------------------------------------------
+---------------------------------------------------------------------------------------------------------------------------------
+John Denver                                                                     Country Roads
+
+(1 rows affected)
+1> exit
+
+...
+
+# Read the sample data from Node 2
```

Comment:
Now, confirm whether the inserted data has been replicated in the secondary nodes.